integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportInvalidAddressTestCase.java

```diff
@@ -35,16 +38,18 @@
  */
 public class MailToTransportInvalidAddressTestCase extends ESBIntegrationTest {
 
-    private static LogViewerClient logViewerClient;
+    private static CarbonLogReader carbonLogReader;
 
     @BeforeClass(alwaysRun = true)
     public void initialize() throws Exception {
         super.init();
-        loadESBConfigurationFromClasspath(
-                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "mailTransport"
-                        + File.separator + "mailTransportReceiver" + File.separator
-                        + "mail_transport_invalid_address.xml");
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        OMElement mailToProxyOMElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(
+                getESBResourceLocation() + File.separator + "mailTransport" + File.separator +
+                        "mailTransportReceiver" + File.separator + "mail_transport_invalid_address.xml")));
+        Utils.deploySynapseConfiguration(mailToProxyOMElement,
```

Comment:
+ should precede in the new line according to the followed code style. Maybe the IDE config is not correct.

Reply:
Done in #575